templates/mod/security.conf.erb

```diff
@@ -54,13 +54,13 @@
     <%- end -%>
     SecArgumentSeparator &
     SecCookieFormat 0
-<%- if scope['os::family'] == 'Debian' -%>
+<%- if scope.lookupvar('::osfamily') == 'Debian' -%>
```

Comment:
I think you can also use `scope['facts']['os']['family']` if you want to use modern facts.